dallinger/command_line/docker_ssh.py

```diff
@@ -75,6 +75,11 @@
     {tls}
 }}
 
+http://logs.{host} {{
```

Comment:
This should be https

Reply:
For consistency with line 73 I think it would be better to omit the protocol and use `logs.{host}`.

Reply:
I removed the scheme bit from the Caddy configuration.